db/deleteItem.go

```diff
@@ -0,0 +1,28 @@
+package db
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+)
+
+func (d *dynamoDB) DeleteItem(listID *string, itemID *string) error {
+	input := &dynamodb.DeleteItemInput{
+		Key: map[string]*dynamodb.AttributeValue{
+			"ListId": {S: listID},
```

Comment:
If listID and itemID weren't pointers you could use `&listID` and `&itemID` here
That way you don't have to worry about them maybe being nil